components/file_system/src/metrics.rs

```diff
@@ -49,4 +49,13 @@ lazy_static! {
             &["type", "op"],
             exponential_buckets(1.0, 2.0, 22).unwrap() // max 4s
         ).unwrap();
+
+    pub static ref RATE_LIMITER_REQUEST_WAIT_DURATION: HistogramVec = register_histogram_vec!(
+            "tikv_rate_limiter_request_wait_duration",
+            "Bucketed histogram of IO rate limiter request wait duration",
+            &["type"],
+            // 0 ~ 1s, 50ms resolution
+            linear_buckets(0.0, 0.05, 22).unwrap()
```

Comment:
I prefer linear because wait duration is bounded within `refill_period`, there is no need to sacrifice resolution for larger range. Speaking of which I might reconfigure this bucket again since `refill_period` has been updated to finer grain.

Reply:
Changing to exponential since we record total-wait now.